gui/src/3D/controls/holographicButton.ts

```diff
@@ -225,32 +224,30 @@ export class HolographicButton extends Button3D {
     private _rebuildContent(): void {
         this._disposeFacadeTexture();
 
-        if (DomManagement.IsDocumentAvailable() && !!document.createElement) {
-            let panel = new StackPanel();
-            panel.isVertical = true;
-
-            if (this._imageUrl) {
-                let image = new Image();
-                image.source = this._imageUrl;
-                image.paddingTop = "40px";
-                image.height = "180px";
-                image.width = "100px";
-                image.paddingBottom = "40px";
-                panel.addControl(image);
-            }
+        let panel = new StackPanel();
+        panel.isVertical = true;
+
+        if (this._imageUrl) {

```

Comment:
Do folks feel we should still gate images around the document existing since there is no image support for babylonnative and babylonreactnative?

Reply:
we will need to wrap images or we can see crashes still.